src/main/java/edu/utdallas/davisbase/representation/UpdateCommandRepresentation.java

```diff
@@ -0,0 +1,62 @@
+package edu.utdallas.davisbase.representation;
+
+import java.util.List;
+
+public class UpdateCommandRepresentation implements CommandRepresentation {
+
+  private String command;
+  private String table;
+  private List<String> columns;
+  private List<String> values;
```

Comment:
```suggestion
  private final List<String> values;
```

Also, instead of a list of strings, this should be a list of literal value expression objects. Otherwise, the compiler will just have to parse the strings into their respective literal value types all over again.